docs/jax.test_util.rst

```diff
@@ -0,0 +1,16 @@
+``jax.test_util`` module
+===================
```

Comment:
This is giving a build warning: `/home/docs/checkouts/readthedocs.org/user_builds/jax/checkouts/26072/docs/jax.test_util.rst:2: WARNING: Title underline too short.`

Reply:
fixed